server/server.js

```diff
@@ -0,0 +1,7 @@
+const listener = require('./app');
+const port = process.env.PORT || 5000;
+
+listener.listen(port, () => {
+    // eslint-disable-next-line no-console
+    console.log(`Listening on ${port}`);
+});
```

Comment:
just curious about the name change?

Reply:
I made the server file `server.js` because Jest was erroring since the `listener.listen()` function does not return after a test is run

Reply:
ok!